packages/web-client/src/webdav/constants/dav.ts

```diff
@@ -28,6 +28,7 @@ export abstract class DavProperty {
   static readonly MimeType: string = '{DAV:}getcontenttype'
   static readonly ResourceType: string = '{DAV:}resourcetype'
   static readonly DownloadURL: string = '{http://owncloud.org/ns}downloadURL'
+  static readonly Highlights: string = '{http://owncloud.org/ns}preview'
```

Comment:
```suggestion
  static readonly Highlights: string = '{http://owncloud.org/ns}highlights'
```
as soon changed in backend